internal/tiger/cmd/mcp_install.go

```diff
@@ -127,8 +143,12 @@ var supportedClients = []clientConfig{
 			"~/Library/Application Support/Code/User/mcp.json",
 			"~/AppData/Roaming/Code/User/mcp.json",
 		},
-		buildInstallCommand: func(tigerPath string) []string {
-			return []string{"code", "--add-mcp", fmt.Sprintf(`{"name":"%s","command":"%s","args":["mcp","start"]}`, mcp.ServerName, tigerPath)}
+		buildInstallCommand: func(serverName, command string, args []string) ([]string, error) {
+			argsJSON, err := json.Marshal(args)
+			if err != nil {
+				return nil, fmt.Errorf("failed to marshal args: %w", err)
+			}
+			return []string{"code", "--add-mcp", fmt.Sprintf(`{"name":"%s","command":"%s","args":%s}`, serverName, command, argsJSON)}, nil
 		},
```

Comment:
Nit: might be a little cleaner to create a map that represents the entire JSON structure, and then marshal that, instead of just marshaling the args and then using string interpolation to create the JSON object. Something like this (not tested):

```suggestion
		buildInstallCommand: func(serverName, command string, args []string) ([]string, error) {
			j, err := json.Marshal(map[string]any{
				"name": serverName,
				"command": command,
				"args": args,
			})
			if err != nil {
				return nil, fmt.Errorf("failed to marshal args: %w", err)
			}
			
			return []string{"code", "--add-mcp", string(j)}, nil
		},
```